build-parent/pom.xml

```diff
@@ -719,6 +720,18 @@
                         </execution>
                     </executions>
                 </plugin>
+		<plugin>
+		    <groupId>org.owasp</groupId>
+		    <artifactId>dependency-check-maven</artifactId>
+		    <version>${owasp-dependency-check-plugin.version}</version>
+		    <executions>
+		        <execution>
+			    <goals>
+			        <goal>check</goal>
+			    </goals>
+			</execution>
+		    </executions>
```

Comment:
The executions could be omitted, since we are explicitly invoking the goal from the command line or the profile.